apps/codelab/src/app/components/buttons-nav-bar/logout-widget/logout-widget.component.html

```diff
@@ -0,0 +1,5 @@
+<button 
```

Comment:
Actually I just realized there is an existing component called `codelab-login-widget` which we could reuse here.

It also already opens a menu, which allows us adding more functionality in the future.